### PR TITLE
[GH-1036] Support readonly flag in URLProperty

### LINKED ISSUE #1036: Bug: URL properties on Kanban view cards have a white background on hover
**Summary:**

**Steps to reproduce the behavior:**
1. Open a board with cards that have a URL property
2. Click Properties, and enable that URL property to be displayed in the preview
3. Hover over a card
4. The URL property value has a white background (on the default theme)

**Expected behavior:**
The background should be transparent (i.e. same gray as the hover style)

**Screenshots:**
URL property hover

**Platform:**
 - v0.9.0 RC6 on Community
 - Chrome 91 on macOS

### PR DESCRIPTION
**Pull request by kamre**
#### Summary
Added readonly flag to `URLProperty` that is passed to `Editable` component.

#### Ticket Link
Fixes #1036

### CODE REVIEW

File: webapp/src/components/properties/link/link.tsx [hunk 1]
```diff
@@ -37,6 +38,7 @@ const URLProperty = (props: Props): JSX.Element => {
                 className='octo-propertyvalue'
                 placeholderText='Empty'
```
Comment (chenilim) [inline webapp/src/components/properties/link/link.tsx:39]: If the URLProperty is readonly, the placeholder should be blank. If you like, go ahead and make that change with this PR, or we can create a separate issue to track it.

**Review by chenilim — APPROVED** [review]
Thanks @kamre! LGTM!

File: webapp/src/components/properties/link/link.tsx (hunk 1 above)
Comment (kamre) [reply webapp/src/components/properties/link/link.tsx]: @chenilim this was already changed in #1008

**Comment by chenilim** [comment]
Thanks again @kamre! @harshilsharma63, please review and merge if it looks good to you. Thanks.

**Review by harshilsharma63 — APPROVED** [review]

**Comment by harshilsharma63** [comment]
@kamre can you please fix the breaking test?

**Comment by kamre** [comment]
@harshilsharma63 I have force pushed the same merge commit.

It seems that we have a [flaky unit test](https://github.com/mattermost/focalboard/blob/main/webapp/src/components/cardDetail/cardDetailContents.test.tsx#L99) that produces different snapshot.

I have already noticed that sometimes there can be a difference for snapshot of this test while updating snapshots for other changes not related to `CardDetailContents`

**Comment by harshilsharma63** [comment]
@kamre I think it produces different results during `npm run updatesnapshot` and `npm run test`.  I've created #1077 to track this.

**Pull request merged**